Gather and report CPLD version and more

dustinmiller opened the pull request:
## Description

Lets add racadm and ipmicfg and update packet-hardware.

## Why is this needed

Need these so we can gather the CPLD version and more from Dell and Supermicro equipment

## How Has This Been Tested?
Tested from within OSIE container and on several Ubuntu 16 machines
Corresponds with packethost/packet-hardware/pull/1


## How are existing users impacted? What migration steps/scripts do we need?
No break. Only fix. 🤞

Commit:
osie-dockerfile: bump packet-hardware version

Newer version of packet-hardware includes the logic to check for
CPLD version for both Dell and Supermicro

Signed-off-by: Dustin Miller <dustin@packet.com>

Commit:
osie-dockerfile: add ipmicfg

Gathered from https://www.supermicro.com/wftp/utility/IPMICFG/
Specifically IPMICFG_1.31.1_build.200623.zip

Signed-off-by: Dustin Miller <dustin@packet.com>

mmlb commented on docker/Dockerfile:
```diff
@@ -74,7 +74,7 @@ RUN apt-get update -y && \
     rm -rf /var/lib/apt/lists/* /tmp/osie/
 
-ARG PACKET_HARDWARE_COMMIT=4702a57d31a86c12ae6b016a88edcc15ee64a42d
+ARG PACKET_HARDWARE_COMMIT=b4e85b9fe707573bcc54954c4f85e620f5c8753f
```
Why? (please answer in commit message :D )

dustinmiller replied:
Yup 👍

mmlb commented on docker/Dockerfile:
```diff
@@ -112,6 +112,12 @@ RUN cd /tmp/osie && \
     rm -r /tmp/osie
 
+# IPMICFG
+COPY lfs/ipmicfg /tmp/osie/
```
It'd be good to have in the commit message the url and version of this file

dustinmiller replied:
Added 🙂

mmlb commented on docker/Dockerfile:
```diff
@@ -118,6 +118,22 @@ RUN cd /tmp/osie && \
+COPY dchipm.ini /tmp/osie/
+RUN cd /tmp/osie && \
+    if [ $(uname -m) != 'aarch64' ]; then \
+      apt-get update && apt-get install -y alien && \
```
should `purge alien` when done

dustinmiller replied:
Indeed, purging 🧹

mmlb commented on docker/Dockerfile:
```diff
@@ -118,6 +118,23 @@ RUN cd /tmp/osie && \
+      tar -xvf DellEMC-iDRACTools-Web-LX-9.4.0-3732_A00.tar.gz && \
+      alien -i iDRACTools/racadm/RHEL8/x86_64/*.rpm && \
+      dpkg -i *.deb && \
+      apt-get purge -y alien && \
```
I think we also need `apt-get autoremove -y`

dustinmiller replied:
Good call, added ✔️

Commit:
osie-dockerfile: add racadm

I found that using the latest debs and the latest rpms both lead to 
build failures. This odd combination results in a successful build. The 
INI seems required and isn't generated otherwise with this installation 
method.

Signed-off-by: Dustin Miller <dustin@packet.com>

Commit:
drone: pin shellcheck to stable

Seems Shellcheck is a moving target on the latest tag, so lets try 
stable and see if this is more fun

Signed-off-by: Dustin Miller <dustin@packet.com>
